to-do-app/src/ToDoApp.css

```diff
@@ -1,28 +1,6 @@
 .Action-Button{
```

Comment:
I see some mix and matching of css selector naming. You can save the work for an upcoming PR, but I would recommend you lower case the css selectors. Here's a few articles on convention:

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

https://css-tricks.com/bem-101/

https://dev.to/xavortm/css-naming-conventions-nen

Whichever route you go, try to make sure you're being consistent.

Reply:
The `ToDoApp.css` is the original CSS styles we created with Pedro. During the refactoring, I'm removing styles from this file and applying new styles. I'm using lower case for naming selectors in `style.css` files. The final goal is to delete this file completely when no one of those CSS selectors is active. 

Thank you for advice!